packages/icons/stories/icon.doc.stories.mdx

```diff
@@ -0,0 +1,103 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Icon, AddDocumentIcon } from "@jpmorganchase/uitk-icons";
+import { withFlexGap } from "docs/decorators/withFlexGap";
+import { ComponentAnatomy } from "docs/components/ComponentAnatomy";
+import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
+
+import {
+  CharacteristicUsage,
+  CSSClassTable,
+} from "css-variable-docgen-components";
+
+<Meta
+  title="Documentation/Icons/Icon"
+  component={Icon}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# Icon
+
+Icon is an important visual representation of an application, a capability, or some other concept or specific entity with meaning on a interface. Icons help users to recognize and understand a concept or an action regardless of linguistic or cultural boundaries.
+
+If an icon is interactive, use [Button](?path=/docs/documentation-core-button--button-variants) (`CTA`, `Primary` or `Secondary` variant).
+
+<Canvas>
+  <Story id="icons-icon--toolkit-icon" />
+</Canvas>
+
+## Use Icon when...
+
+- You need a visual representation of a concept.
+- An icon doesn't need to be interactive.
+
+## Features
+
+We have a default icon set that starts from 12px for high and medium density (MD), then 14px for low density (LD), and 16px for touch density (TD). (Since our base size of high density (HD)/default is 10px but we don’t allow icons size smaller than 12px, hence why here the HD/default will start from 12px, and its 2X is 20px, 3X is 30px etc.)
+You have more size choices by adding multiplies to 2x, 3x, 4x etc.
+
+<Canvas>
+  <Story id="icons-icon--toolkit-icon-multiple-sizes" />
+</Canvas>
+
+We provide two types of icons: default and solid. Some special icons may be excluded from it like `plus` and `loader`.
+
+<Canvas>
+  <Story id="icons-icon--toolkit-types" />
+</Canvas>
+
+To override the default color of the icon, you can use the `color` css variable.
+
+<Canvas>
+  <AddDocumentIcon
+    style={{ "--uitkIcon-color": "var(--uitk-status-info-foreground)" }}
+  />
+</Canvas>
+
+You can supply custom icons as `children` of the `Icon` component by using an `<svg>`. Whenever you add a custom icon you need to pass an `aria-label` to the Icon.
+
+<Canvas>
+  <Story id="icons-icon--custom-svg-icon" />
+</Canvas>
+
+### Icons List
+
+<Canvas>
+  <Story id="icons-icon--all-icons-with-search" />
+</Canvas>
+
+## Configuring Icon
+
+### API
+
+```
+import { Icon } from "@jpmorganchase/uitk-icons";
+```
+
+### Props
+
+<ArgsTable of={Icon} />
+
+### Anatomy
+
+<Canvas>
+  <ComponentAnatomy>
+    <AddDocumentIcon />
+  </ComponentAnatomy>
+</Canvas>
+
+### Classes
+
+<CSSClassTable of={Icon} />
+
+## Characteristics Used
+
+<CharacteristicUsage of={Icon} />
+
+## Accessibility
+
+To ensure that icon-only buttons are ADA compliant, when icon is inside a button you must display the button’s label in a supporting Tooltip that is revealed on mouse hover or keyboard focus.
```

Comment:
I would argue this is not specific to Icon and shouldn't be here. Seems like something that should be on the Button page and potentially linked to